lpcr_donation_tax_report/models/donation_tax_receipt.py

```diff
@@ -0,0 +1,95 @@
+# SPDX-FileCopyrightText: 2025 Coop IT Easy SC
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class DonationTaxReceipt(models.Model):
+    _inherit = "donation.tax.receipt"
+
+    # Remove the domain restriction that partner can't have parent
+    # because we want individuals not just their companies
+    partner_id = fields.Many2one(
+        "res.partner",
+        string="Donor",
+        required=True,
+        ondelete="restrict",
+        index=True,
+        tracking=True,
```

Comment:
what doesn’t change should not be present.
```suggestion
        domain=[]
```